core/src/apps/common/request_pin.py

```diff
@@ -146,7 +140,7 @@ async def error_pin_invalid(ctx: Context) -> NoReturn:
     await show_error_and_raise(
         ctx,
         "warning_wrong_pin",
-        "The PIN you entered is invalid.",
+        "The PIN you have entered is not valid.",
```

Comment:
Cannot find it in `Figma` now, but it comes from the cooperation with @Hannsek 

Yes, it will change some TT screens, it is already accounted for in UI tests - when CI will be working, there will be master-diff